src/main/java/org/openmaptiles/util/OmtLanguageUtils.java

```diff
@@ -65,10 +69,36 @@ public class OmtLanguageUtils {
    * <li>name_int is the first of int_name name:en name:latin name</li>
    * </ul>
    */
+  private static final Logger LOGGER = LoggerFactory.getLogger(OmtLanguageUtils.class);
+  private static List<String> extraNameTags = List.of();
+
   public static Map<String, Object> getNamesWithoutTranslations(Map<String, Object> tags) {
     return getNames(tags, null);
   }
 
+  public static List<String> setExtraNameTags(List<String> extraNameTags) {
+    if (extraNameTags != null) {
+      OmtLanguageUtils.extraNameTags = extraNameTags;
+    }
+    return extraNameTags;
+  }
+
+  private static Map<String, Object> getExtraNameTags(Map<String,Object> tags) {
+    if (!extraNameTags.isEmpty()) {
+      Map<String,Object> extraTags =  extraNameTags.stream()
+        .filter(tags::containsKey)
+        .filter(tag -> tags.get(tag) != null && !tags.get(tag).equals(""))
+        .collect(Collectors.toMap(t -> t, tags::get));
+      if (!extraTags.isEmpty()) {
+        LOGGER.debug("Found {} extra tags from OSM object with name {}", extraTags.size(), tags.get("name"));
+      }
```

Comment:
Once we verify that this is working, we might want to remove this debug log since this could potentially match a lot of times.